module/rdpCapture.c

```diff
@@ -632,72 +604,70 @@ rdpCapture0(rdpClientCon *clientCon,
         (*out_rects)[i] = rect;
     }
 
-    if ((src_format == XRDP_a8r8g8b8) && (dst_format == XRDP_a8r8g8b8))
+    src = id->pixels;
+    dst = id->shmem_pixels;
+    dst_format = clientCon->rdp_format;
+    src_stride = id->lineBytes;
+    dst_stride = clientCon->cap_stride_bytes;
```

Comment:
That depends on the version of xorg you are building against and the system you are building on.  Xorg used to be way behind on there compile requirement.  But you're right, the latest one is using C99 stuff.  I think sprinkling variables allover functions is another issue.